lua/entities/cap_doors_contr.lua

```diff
@@ -102,7 +102,7 @@ function ENT:PressButton()
 				self.Entity:EmitSound(self.Sounds.PressDest,100,math.random(90,110));
 			elseif frame.Lockdown then
 				no_sound = true
-				self.Entity:EmitSound(self.Sounds.LockedDest,100,math.random(90,110));
+				self.Entity:EmitSound(self.Sounds.LockedDest,100,100); -- dont put random pitch, it sounds weird then
```

Comment:
I don't care how it was, its gmod)) And almost everything in cap have such thing, so why not then? Like i said it add feel like it different sounds.